src/bundle/Resources/views/content/modal_location_bulk_operation_failed.html.twig

```diff
@@ -0,0 +1,46 @@
+<div
+    class="modal fade ez-bulk-operation-failed-modal"
+    id="bulk-operation-failed-modal"
+    tabindex="-1"
+    role="dialog">
+    <div
+        class="modal-dialog"
+        role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">
+                    
+                </h5>
+                <button
+                    type="button"
+                    class="close"
+                    data-dismiss="modal"
+                    aria-label="Close">
+                    <svg
+                        class="ez-icon ez-icon--medium"
+                        aria-hidden="true">
+                        <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#discard"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="modal-body">
+                <div class="ez-table-header"></div>
```

Comment:
Seems unused.

Reply:
This table looks bad without this header.

Reply:
I see, but on the mockups there is different table:
<img width="1047" alt="mockup" src="https://user-images.githubusercontent.com/12594013/49725314-b305c480-fc6b-11e8-8240-81967539e799.png">

Reply:
Yes, but during discussing on this feature with PM I've got feedback that we want a different table here. Probably there will be a title but I didn't get final feedback what title do we want.

Reply:
@tischsoic what's the status here? :) is the table state final?